tests/integration/views/test_helpers.py

```diff
@@ -1,16 +1,70 @@
 import os
 
+import pytest
+from django.test import RequestFactory
+from django.template.response import TemplateResponse
+
 from airlock.views import helpers
 
 
-def test_serve_file(tmp_path):
-    path = tmp_path / "test.txt"
-    path.write_text("hi")
+@pytest.mark.parametrize(
+    "suffix,mimetype",
+    [
+        (".html", "text/html"),
+        (".csv", "text/html"),
+        (".png", "image/png"),
+        (".txt", "text/html"),
+    ],
+)
+def test_serve_file_rendered(tmp_path, suffix, mimetype):
+    rf = RequestFactory()
```

Comment:
pytest-django has a `rf` fixture that could be used here instead (and in the other tests in this file)

Reply:
Ah, yes, I'd forgotten about that, will add.